triton-server/src/api/api-routes.ts

```diff
@@ -101,6 +101,23 @@ router.get(
 	})
 )
 
+router.get(
+	'/runs-requests/',
```

Comment:
are runs-requests ? not sure about the name ... staging-requests? the relation between runs and requests is unclear at best.